app/lib/widgets/NewsItem.dart

```diff
@@ -111,4 +71,54 @@ class NewsItem extends StatelessWidget {
       ],
     );
   }
+
+  Widget? _buildImage() {
+    var image = news.image();
+    if (image == null) {
+      return null;
+    }
+    return Image.memory(Uint8List.fromList(image), fit: BoxFit.cover);
```

Comment:
Also could use ```CachedMemoryImage``` here.

Reply:
Fixed it

Reply:
Need to find alternative solutions for caching.